app/services/support_interface/candidate_email_send_counts_export.rb

```diff
@@ -0,0 +1,24 @@
+module SupportInterface
+  class CandidateEmailSendCountsExport
+    def data_for_export
+      emails_with_counts.map do |email_hash|
+        {
+          'Email' => email_hash['mail_template'],
+          'Send count' => email_hash['send_count'],
+          'Last sent at' => email_hash['last_sent_at'],
+        }
+      end
+    end
+
+  private
+
+    def emails_with_counts
+      Email
+        .select(
+          'mail_template, COUNT(mail_template) AS send_count, MAX(created_at) AS last_sent_at',
+        )
+        .group('mail_template')
+        .order('mail_template ASC')
```

Comment:
Makes sense, I guess the only thing this won't reveal is email templates that have never been sent (or rather not sent since we started tracking) but I don't think that matters.

Reply:
That's correct, we've only been logging emails in this table since 21 February 2020.